alcs-frontend/src/app/features/application/review/decision-meeting/decision-meeting.component.html

```diff
@@ -0,0 +1,37 @@
+<br />
```

Comment:
Why is this here? If this is for spacing you should add margin-top to the div below instead.

Reply:
Not all styling needs to be with margins and paddings. I prefer to add <br> to sections whenever I need to have even spacing between sections. This way the spacing is always the same and no need of additional wrappers or css tweaks